packages/fetchai/connections/gym/tests/test_gym.py

```diff
@@ -43,9 +41,6 @@
 
 logger = logging.getLogger(__name__)
 
-CUR_PATH = os.path.dirname(inspect.getfile(inspect.currentframe()))
```

Comment:
A package cannot depend on other code other than other packages which are referenced as dependencies in `{package_type}.yaml`